### PR TITLE
Fixing a typo in KBO Mirror class for DS Bender RMS PV

### PR DESCRIPTION
**Pull request by jyotiphy**
A typo in KBO Mirror class for DS Bender RMS. Broken PV of DS RMS value for RIX and TMO

## Pre-merge checklist
- [x] Code works interactively
- [x] Test suite passes on travis
- [ ] Ran docs/pre-release-notes.sh and created a pre-release documentation page
- [ ] Pre-release docs include context, functional descriptions, and contributors as appropriate

Screen Shot 2021-06-07 at 4 56 21 PM
Screen Shot 2021-06-07 at 4 57 05 PM

### CODE REVIEW

**Comment by ZLLentz** [comment]
Have you tested the fix? I only see the error screenshots. The reason this error exists is because the original PR (8 months ago) that added the KB class didn't get tested enough, so I think we need to test before merging or else we just do this again tomorrow. See previous PR that wasn't tested: https://github.com/pcdshub/pcdsdevices/pull/612

I'm going to add the pre-merge checklist that comes with a new PR back into the pull request text and check that boxes that don't matter for this PR. We need to fill out the pre-merge checklist before merging.

**Comment by jyotiphy** [comment]
I had checked the PVs and "*:ENC:BEND:DS:RMS_RBV" seems to be correct, not the current listed one  "*:ENC:BENDER:DS:RMS_RBV"

**************************
rix-daq:~> caget MR4K2:KBV:ENC:BEND:US:RMS_RBV
MR4K2:KBV:ENC:BEND:US:RMS_RBV  9.33684e-05
rix-daq:~> caget MR4K2:KBV:ENC:BEND:DS:RMS_RBV
MR4K2:KBV:ENC:BEND:DS:RMS_RBV  2.90676e-05
rix-daq:~> caget MR4K2:KBV:ENC:BENDER:DS:RMS_RBV
Channel connect timed out: 'MR4K2:KBV:ENC:BENDER:DS:RMS_RBV' not found.
rix-daq:~> caget MR3K2:KBH:ENC:BEND:DS:RMS_RBV
MR3K2:KBH:ENC:BEND:DS:RMS_RBV  2.58716e-05
rix-daq:~> caget MR3K2:KBH:ENC:BENDER:DS:RMS_RBV
Channel connect timed out: 'MR3K2:KBH:ENC:BENDER:DS:RMS_RBV' not found.

**Comment by ZLLentz** [comment]
Okay, that looks good to me. Do you intend to run the code you've edited and update the pre-release notes? Or are you asking me to do the rest?

**Comment by jyotiphy** [comment]
Yes, please! I am not sure what are the steps to release it and propagate it to the hutch environment.

**Comment by ZLLentz** [comment]
Ok, I'll do the merge/documentation steps tomorrow and put this fix into dev right now. I won't merge a PR that's not fully documented but there's no reason we have to merge this before applying it to the screens.

**Comment by ZLLentz** [comment]
Confirmation that this works in the wild, currently applied to dev (all the screens still run dev)

image

**Review by nrwslac — APPROVED** [review]

**Comment by ZLLentz** [comment]
Merging as-is and doing the docs in a follow-up since this was made on a master branch

**Pull request merged**
